utils/write-dependencies.js

```diff
@@ -78,7 +71,16 @@ module.exports = (projectPath, inLernaProject) => {
 
   // We don't need to install these for root lerna packages. They will be installed for the packages.
   if (isMultiPackageProject(projectPath) && !inLernaProject) {
-    return;
+    if (added.length > 0) {
+      pjson.actions.push(`adding required devDependencies ${added.join(', ')}`);
+    }
+
+    if (removed.length >= 0) {
+      pjson.actions.push('removed devDependencies controlled by dev-scripts');
+    }
+
+    pjson.write();
+    return added.length > 0;
```

Comment:
[This commit](https://github.com/forcedotcom/sfdx-dev-packages/commit/2adecbf62af52ad40d1ce737c4cb313541d939ac#diff-2a67ec59130a57e4d4e8a872ce722e0b760223f258453faf70528c8bbfc9c9b8R65-R68) introduced some logic that broke updating deps in lerna repos (there was a `pjson.write` at the bottom of the file, after that commit it never wrote lerna's root package.json file).